README.md

```diff
@@ -1,33 +1,28 @@
 # Intercom ([docs](https://dbt-intercom.netlify.app/))
+> NOTE: Our Intercom [model](https://github.com/fivetran/dbt_intercom) and [source](https://github.com/fivetran/dbt_intercom_source) dbt packages only work with connectors that were [created in July 2020](https://fivetran.com/docs/applications/intercom/changelog) or later. If you created your connector before July 2020, you must set up a new Intercom connector to use these dbt packages.
 
-This package models [Intercom] data from [Fivetran's connector](https://fivetran.com/docs/applications/intercom). It uses data in the format described by [this ERD](https://docs.google.com/presentation/d/1K3HTGqNQ-neUNeTtjJq42RHBV68_4FuXFp8X81zJ5Xo/edit#slide=id.p).
+This package models Intercom data from [Fivetran's connector](https://fivetran.com/docs/applications/intercom). It uses data in the format described by [this ERD](https://docs.google.com/presentation/d/1K3HTGqNQ-neUNeTtjJq42RHBV68_4FuXFp8X81zJ5Xo/edit#slide=id.p).
 
-The main focus of this package is to enable users to better understand and recognize their responsiveness and effectiveness in relation to 
-customer conversations via Intercom. You can easily gain insights from multiple conversations
-with customers to determine and discover how your admins are performing, customer sentiment from conversations via the Intercom ranking functionality, 
-and gain valuable information through leveraging the contact and company models with conversations.
-
-## Compatibility
-> Please be aware the [dbt_intercom](https://github.com/fivetran/dbt_intercom) and [dbt_intercom_source](https://github.com/fivetran/dbt_intercom_source) packages will only work with the [Fivetran Intercom connector](https://fivetran.com/docs/applications/intercom/changelog) which was released in July 2020, or any version thereafter. If your Intercom connector was set up prior to the July 2020 release, you will need to set up a new Intercom connector in order for the Fivetran dbt Intercom packages to work.
+This package enables you to better understand your responsiveness and effectiveness in customer conversations.
```

Comment:
@fivetran-joemarkiewicz Can you elaborate on what the output of this dbt package is? Here's an example from the Jira dbt package's README:
```
This package enables you to better understand the workload, performance, and velocity of work done by your team using Jira issues. It achieves this by:
- Creating a daily issue history table to enable the quick creation of agile reports, such as burndown charts, along any issue field
- Enriching the core issue table with relevant data regarding its workflow and current state
- Aggregating bandwidth and issue velocity metrics along projects and users
```

Reply:
See below for suggested update
```
This packages enables you to better understand the performance, responsiveness, and effectiveness of your teams conversations with customers via Intercom. It achieves this by:
- Creating an enriched and aggregated conversations table to enable large scale reporting on all conversations in progress and closed.
- Enriching conversation data with a corresponding contacts table which provides additional contact information and tags that can be used to gain insights regarding who conversations are being held with.
- Aggregating your teams performance across all conversations in a table that allows for quick visibility of team members overall conversation performance.
- Providing aggregate rating and timeliness metrics for all conversations with customers associated with companies logged within Intercom which enables conversation performance reporting at a company level.
```

Reply:
Thanks for this, @fivetran-joemarkiewicz ! I've simplified the language a bit to improve readability and added it to the README. Can you please check my edit to make sure that it's accurate?

Reply:
The edits you pushed to the readme are great, thank you!